tests/timeseries_util.erl

```diff
@@ -41,101 +59,91 @@
 -define(MAXFLOAT,            math:pow(2, 63)).
 -define(MULTIPLECLUSTERSIZE, 3).
 
-confirm_create(ClusterType, DDL, Expected) ->
+-spec(confirm_create(atom(), string()) -> {ok, string()} | term()).
+confirm_create(ClusterType, DDL) ->
```

Comment:
@macintux I'm open to renaming suggestions. We could use your favorite, `verify_put`. :trollface:  Or `test_put` or `ts_put`?